src/controllers/index.mjs

```diff
@@ -0,0 +1,187 @@
+import Router from '@koa/router'
+import { initialize } from 'koa-openapi'
+import cors from '@koa/cors'
+import koaBody from 'koa-body'
+import bytes from 'bytes'
+import path from 'path'
+
+import apiDoc from './api-doc.js'
```

Comment:
Not for this PR, and it was a problem beforehand, but this does feel like a very inefficient way to load everything, but while I was trying to fix the open-api thing this is what I found that actually loads everything correctly. I'm sure there's a better way, so if you think of something cleaner, all ears!

Reply:
You mean importing all the different endpoints? Or this specific api-dock.js import? :sweat_smile:

Reply:
oh haha all the different endpoints, not just this one.